examples/00-EDB/02_edb_to_ipc2581.py

```diff
@@ -46,12 +46,12 @@
 )
 
 ###############################################################################
-# Create IPC2581 file
+# Create ipc2581 file
 # ~~~~~~~~~~~~~~~~~~~
-# Create the IPC2581 file.
+# Create the ipc2581 file.
 
 edb.export_to_ipc2581(ipc2581_file, "inch")
-print("IPC2581 File has been saved to {}".format(ipc2581_file))
+print("ipc2581 File has been saved to {}".format(ipc2581_file))
```

Comment:
```suggestion
print("IPC2581 file has been saved to {}".format(ipc2581_file))
```
There should be a period at the end of the message string, but I wasn't sure exactly where to put it.